Source/VerveAdapter.swift

```diff
@@ -68,13 +68,14 @@ final class VerveAdapter: PartnerAdapter {
     /// - parameter request: Information about the ad load request.
     /// - parameter completion: Closure to be performed with the fetched info.
     func fetchBidderInformation(request: PreBidRequest, completion: @escaping ([String : String]?) -> Void) {
-        guard let appToken else {
-            let error = error(.prebidFailureInvalidArgument, description: "App token is empty")
+        let signalData = HyBid.getCustomRequestSignalData("cb")
+        guard let signalData else {
+            let error = error(.prebidFailureInvalidArgument, description: "Signal data is empty")
             log(.fetchBidderInfoFailed(request, error: error))
             completion(nil)
             return
         }
-        completion(["app_auth_token": appToken])
+        completion(["signal_data": signalData])
```

Comment:
May want to sync with the backend team to make sure this change is expected

Reply:
@ricealexanderb Did you already sync with the backend team on this?  This was your PR and am just trying to get it to pass checks.

Reply:
There was at least one round of coordination with the backend but I don't remember if we ever got it resolved. I'm looking into it now.

Reply:
@bwised I added you to the old slack channel so you could see this thread: [](https://chartboost.slack.com/archives/C04RH1CH92L/p1697819918007689)
Looks like the backend is piping `signal_data` along to verve.